app/services/partners/family_request_create_service.rb

```diff
@@ -52,26 +54,25 @@ def valid?
 
     def item_requests_attributes
       @item_requests_attributes ||= family_requests_attributes.map do |fr_attr|
-        item = included_items.find { |i| i.id == fr_attr[:item_id].to_i }
-
         {
-          item_id: item&.id,
-          quantity: convert_person_count_to_item_quantity(item_id: fr_attr[:item_id], person_count: fr_attr[:person_count])&.to_i
+          item_id: fr_attr[:item_id],
+          quantity: convert_person_count_to_item_quantity(item_id: fr_attr[:item_id], person_count: fr_attr[:person_count])&.to_i,
+          children: fr_attr[:children]
```

Comment:
```suggestion
          child: fr_attr[:child]
```
Does it make sense if we make this singular? Since each item is going to a single child and not children?

Reply:
Hm, I may have misunderstood the data model but I think it is plural here because

`Partners::ItemRequest`  has many children through `Partners::ChildItemRequest`s https://github.com/rubyforgood/diaper/blob/72e64cb6b3171d0d499b860fb6858923ee02fc15/app/models/partners/item_request.rb#L17-L18

e.g. if 2 kids both request "Kids (Size 6)", we'll make a single `Partners::ItemRequest` for that item and then 2 `Partners::ChildItemRequest`s, one for each kid. This is the scenario I tested in the service spec https://github.com/rubyforgood/diaper/pull/2278/files#diff-6934bb9b3798c06f69cc8f0cff58268d3b0485a50e82fb120f394eba1976a94bR45-R86

Reply:
Ahh, thanks for looking into this. Yes, this was also a bit unclear to me as well. If this is the behavior as it is was then I don't think it makes sense to change it here. We can revisit this when we want to change functionality.